compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -4608,7 +4608,8 @@ static bool isBitwiseLongComplement(TR::Node * n)
 template <typename T>
 static bool checkAndReplaceRotation(TR::Node *node,TR::Block *block, TR::Simplifier *s)
    {
-   if (feGetEnv("TR_DisableROLSimplification"))
+   static char* env = (feGetEnv("TR_DisableROLSimplification"));
```

Comment:
Rather than `env`, why not call the auto `disableROLSimplification` to make it explicit?